rust/cache/src/foyer.rs

```diff
@@ -343,6 +370,7 @@ where
                     .with_file_size(config.file_size * MIB),
             )
             .with_flush(config.flush)
+            .with_recover_mode(foyer::RecoverMode::Strict)
```

Comment:
switching to this by default

Reply:
OOC, what does this do? Should we also add comments?

Reply:
I think the foyer documentation is plenty? It makes recovery panic if it fails, rather than masking errors